src/commands/a32nx/custom-waypoint.ts

```diff
@@ -0,0 +1,18 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
+export const customWaypoint: CommandDefinition = {
+    name: ['waypoint', 'customwaypoint'],
+    description: 'Provides a link to the a32nx custom waypoint guide',
+    category: CommandCategory.A32NX,
+    executor: async (msg) => {
+        const waypointEmbed = makeEmbed({
+            title: 'FlyByWire A32NX | Custom Waypoints',
```

Comment:
Maybe better to retool this to just "stored waypoints" instead and link the start of the guide.